clients/ts/package.json

```diff
@@ -5,21 +5,45 @@
   "main": "index.js",
   "scripts": {
     "build": "cd ./signalr && npm run build && cd ../signalr-protocol-msgpack && npm run build",
-    "test": "cd ./signalr && npm run test && cd ../signalr-protocol-msgpack && npm run test"
+    "test": "jest"
   },
   "author": "Microsoft",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@types/jasmine": "^2.8.3",
+    "@types/jest": "^22.2.3",
     "@types/node": "^8.5.2",
-    "jasmine": "^2.8.0",
+    "jest": "^22.4.3",
     "rimraf": "^2.6.2",
     "rollup": "^0.53.4",
     "rollup-plugin-commonjs": "^8.2.6",
     "rollup-plugin-node-resolve": "^3.0.2",
     "rollup-plugin-sourcemaps": "^0.4.2",
+    "ts-jest": "^22.4.4",
     "tslint": "^5.9.1",
     "typescript": "^2.7.1",
     "uglify-js": "^3.3.5"
+  },
+  "jest": {
+    "globals": {
+      "ts-jest": {
+        "tsConfigFile": "./tsconfig.jest.json"
+      }
+    },
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest"
+    },
+    "testEnvironment": "node",
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
```

Comment:
This is The Official Regex for `ts-jest`, and it's just an update to The Official Regex for `jest` to support `ts` and `tsx` files. The patterns are:

* Any file in a `__tests__` subfolder anywhere in the project directory.
* Any `.js`, `.jsx`, `.ts`, or `.tsx` file with the suffix `.test` or `.spec` anywhere in the project directory.

Reply:
I would say add a comment but JSON so yeah...

Reply:
Yeah... There are tricks to use, since most of the package.json settings and subsettings ignore unknown properties (`"_comment": "A comment"`), but I don't think it really works. I may throw a README.md in `clients/ts` and put some notes in there.